dronegen/tag.go

```diff
@@ -0,0 +1,425 @@
+package main
+
+import (
+	"fmt"
+	"strings"
+)
+
+const (
+	// rpmPackage is the RPM package type
+	rpmPackage = "rpm"
+	// debPackage is the DEB package type
+	debPackage = "deb"
+)
+
+type tagBuildType struct {
+	os      string
+	arch    string
+	fips    bool
+	centos6 bool
+}
+
+// tagMakefileTarget gets the correct tag pipeline Makefile target for a given arch/fips combo
+func tagMakefileTarget(params tagBuildType) string {
+	makefileTarget := fmt.Sprintf("release-%s", params.arch)
+	if params.centos6 {
+		makefileTarget += "-centos6"
+	}
+	if params.fips {
+		makefileTarget += "-fips"
+	}
+	return makefileTarget
+}
```

Comment:
can this be combined with `pushBuildType`, if `push.go` just never sets `params.centos6` to `true`?